packages/common/util.js

```diff
@@ -7,13 +7,11 @@ const os = require('os');
 /**
  * Synchronously makes a temporary directory, smoothing over the differences between
  * mkdtempSync in node.js for various platforms and versions
+ * 
  * @param {string} name - A base name for the temp dir, to be uniquified for the final name
- * @return - The absolute path to the created dir
+ * @returns {string} - The absolute path to the created dir
  */
 exports.mkdtempSync = (name) => {
```

Comment:
As far as I can tell, this function is never used.  Instead of modifying it, consider removing it altogether.

Reply:
I am using it... that's how I found out that it did not work on AWS and needed to be fixed.

Reply:
Have you committed those tests?  I've searched all through the code and have not found any instance where that function is being used.

```
$ ag mkdtempSync
packages/common/util.js
9: * mkdtempSync in node.js for various platforms and versions
14:exports.mkdtempSync = (name) => {

packages/deployment/test/test_adapter.js
14:  t.context.temp = fs.mkdtempSync(`${os.tmpdir()}${path.sep}`);

packages/deployment/test/test_lambda.js
16:  t.context.temp = fs.mkdtempSync(`${os.tmpdir()}${path.sep}`);
```

Reply:
Clarified with @cappelaere that this is being used in the gitc-deploy repo.